deps.edn

```diff
@@ -0,0 +1,6 @@
+{:paths   ["src"]
+ :deps    {org.clojure/clojure         {:mvn/version "1.10.1"}
+           org.clojure/tools.namespace {:mvn/version "1.0.0"}
+           integrant/integrant         {:mvn/version "1.0.0"}}
+ :aliases {:test {:extra-deps  {org.clojure/clojurescript {:mvn/version "1.10.773"}}
```

Comment:
As this is a PR against `master`, the ClojureScript part should be removed for now.

Reply:
Removed aliases.

Reply:
Can you fix the spacing too?

Reply:
Fixed spacing.